.travis.yml

```diff
@@ -15,4 +15,18 @@ before_script:
   - npm install
 script:
   - gulp build
-  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && sh deploy.sh || true'
\ No newline at end of file
+  - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && sh deploy.sh || true'
+
+after_success:
+  .cicd/beta_npm_ver.js
+
+deploy:
+  skip_cleanup: true
+  provider: npm
+  email:
```

Comment:
@temujin @angieskazka @jheeffer 
email and api_key has to be set before merging